hrreport/tests/phpunit/CRM/HRReport/Form/Contact/HRSummaryTest.php

```diff
@@ -1,50 +1,22 @@
 <?php
-/*
- +--------------------------------------------------------------------+
- | CiviHR version 1.4                                                 |
- +--------------------------------------------------------------------+
- | Copyright CiviCRM LLC (c) 2004-2014                                |
- +--------------------------------------------------------------------+
- | This file is a part of CiviCRM.                                    |
- |                                                                    |
- | CiviCRM is free software; you can copy, modify, and distribute it  |
- | under the terms of the GNU Affero General Public License           |
- | Version 3, 19 November 2007 and the CiviCRM Licensing Exception.   |
- |                                                                    |
- | CiviCRM is distributed in the hope that it will be useful, but     |
- | WITHOUT ANY WARRANTY; without even the implied warranty of         |
- | MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.               |
- | See the GNU Affero General Public License for more details.        |
- |                                                                    |
- | You should have received a copy of the GNU Affero General Public   |
- | License and the CiviCRM Licensing Exception along                  |
- | with this program; if not, contact CiviCRM LLC                     |
- | at info[AT]civicrm[DOT]org. If you have questions about the        |
- | GNU Affero General Public License or the licensing of CiviCRM,     |
- | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
- +--------------------------------------------------------------------+
-*/
 
-require_once 'CiviTest/CiviReportTestCase.php';
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+
 /**
  *  Test report outcome
  *
  * @package CiviCRM
+ * @group headless
  */
-class CRM_HRReport_Form_Contact_HRSummaryTest extends CiviReportTestCase {
-  protected $_tablesToTruncate = array(
-    'civicrm_contact',
-    'civicrm_email',
-    'civicrm_phone',
-    'civicrm_address',
-    'civicrm_hrjob',
-    'civicrm_hrjob_health',
-    'civicrm_hrjob_hour',
-    'civicrm_hrjob_leave',
-    'civicrm_hrjob_pay',
-    'civicrm_hrjob_pension',
-    'civicrm_hrjob_role'
-  );
+class CRM_HRReport_Form_Contact_HRSummaryTest extends CiviReportTestCase implements HeadlessInterface , TransactionalInterface {
+
+  public function setUpHeadless() {
+    return \Civi\Test::headless()
+      ->installMe(__DIR__)
+      ->install('org.civicrm.hrabsence')
```

Comment:
this is an old extension so no surprise it's depends on hrjob extension which is no longer needed since it get replaced with hrjobcontract + hrjobrole ... I can't enable hrjob role here because it may cause some other issues and its not needed anymore so we can't depends on it .. and that's maybe why running the tests for this extension failed ... I created a new task to deal with these issues and going to investigate them further there .